lib/auth/machineid/workloadidentityv1/decision_test.go

```diff
@@ -26,6 +27,55 @@ import (
 	"github.com/gravitational/teleport/api/types"
 )
 
+func Test_decided(t *testing.T) {
```

Comment:
The most nitpicky nitpick 😅 

```suggestion
func Test_decide(t *testing.T) {
```